core/bin/system-constants-generator/Cargo.toml

```diff
@@ -16,8 +16,8 @@ zksync_types = { path = "../../lib/types" }
 zksync_utils = { path = "../../lib/utils" }
 zksync_contracts = { path = "../../lib/contracts" }
 multivm = { path = "../../lib/multivm" }
-zkevm_test_harness_1_3_3 = { git = "https://github.com/matter-labs/era-zkevm_test_harness.git", branch = "v1.3.3", package = "zkevm_test_harness" }
 
+zkevm_opcode_defs = { git = "https://github.com/matter-labs/era-zkevm_opcode_defs.git", branch = "v1.4.1"}
```

Comment:
you should be able to import it from `multivm::zk_evm_latest`

Reply:
ack. good idea.